python/lsst/ip/isr/ptcDataset.py

```diff
@@ -440,6 +440,9 @@ def fromTable(cls, tableList):
         inDict['badAmps'] = []
         inDict['photoCharge'] = dict()
 
+        calibVersion = metadata['PTC_VERSION']
```

Comment:
Do you need a check on `calibVersion` for this warning message?  It seems like that message should only trigger if `calibVersion == 1.0`.

Reply:
Yes, good catch, thanks. I had the message inside the loop, with the check, but when I decided to just have one message outside the loop, I did not put a check.